.travis.yml

```diff
@@ -1,14 +1,14 @@
 language: generic
-sudo: required
 dist: bionic
+os: linux
 
 services:
   - docker
 
 script:
   - docker build --pull -t riotdocker .
   - docker image ls riotdocker:latest
-  - git clone --depth 1 https://github.com/RIOT-OS/RIOT -b 2020.04-branch
+  - git clone --depth 1 https://github.com/RIOT-OS/RIOT -b 2020.10-branch
```

Comment:
Can `2020.10-branch` be a variable so we only have to change it in once place?

Reply:
yup, done!